### PR TITLE
Add bidhandshakecycle model

### PR DESCRIPTION
**Pull request by mjoyce91**

### CODE REVIEW

**Comment by mjoyce91** [comment]
@scott062 - not done, but tagging you for awareness

File: talentmap_api/fsbid/services/common.py [hunk 1]
```diff
@@ -169,12 +169,18 @@ def sorting_values(sort):
 
 
 def get_results(uri, query, query_mapping_function, jwt_token, mapping_function, api_root=API_ROOT):
-    url = f"{api_root}/{uri}?{query_mapping_function(query)}"
+    if query_mapping_function and query:
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/common.py:172]: nice safety check

File: talentmap_api/bidding/views/bidhandshakecycle.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    def perform_create(self, serializer):
+        in_group_or_403(self.request.user, "superuser")
+        instance = serializer.save()
+
+    def perform_update(self, serializer):
+        in_group_or_403(self.request.user, "superuser")
+        instance = serializer.save()
+    
+    def perform_delete(self, serializer):
+        in_group_or_403(self.request.user, "superuser")
```
Comment (elizabeth-jimenez) [inline talentmap_api/bidding/views/bidhandshakecycle.py:50]: Do we want to also expose for `bidcycle_admin `? On FE we expose BidCycles menu for them

Comment (mjoyce91) [reply talentmap_api/bidding/views/bidhandshakecycle.py:50]: I think for first pass, just want to keep it limited to our superusers

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍬 👍

**Pull request merged**